vm/vmStmt.go

```diff
@@ -618,6 +618,99 @@ func (runInfo *runInfoStruct) runSingleStmt() {
 		}
 		runInfo.rv = nilValue
 
+	// SelectStmt
+	case *ast.SelectStmt:
+		e := runInfo.env
+		runInfo.env = e.NewEnv()
+
+		body := stmt.Body.(*ast.SelectBodyStmt)
+		letsExprs := []ast.Expr{nil}
+		bodies := []ast.Stmt{nil}
```

Comment:
Do not think these slices are needed.  Could a index be used to lookup in the Select Statment?